conans/client/command.py

```diff
@@ -746,7 +746,9 @@ def imports(self, *args):
         return self._conan.imports(args.path, args.import_folder, args.install_folder)
 
     def export_pkg(self, *args):
-        """Exports a recipe, then creates a package from local source and build folders.
+        """
```

Comment:
For me is almost the same confusing. Could both paragraphs be merged into something like:

```
Exports a recipe and creates a binary package from --source-folder and --build-folder by calling the package() method or, if --package-folder is specified, copying directly the files.
```

Reply:
> Exports a recipe and create a binary package from local files.
>
> If '--package-folder' is provided it will copy the files from there, otherwise it will execute package() method over '--source-folder' and '--build-folder' to create the binary package.

??

Reply:
Could you elaborate on your question?

Reply:
Not a question, a proposal for the help text.

Reply:
Oh! I got confused with the format. Sure. It sounds better.